packages/shared/src/components/post/write/ShareLink.tsx

```diff
@@ -51,19 +68,34 @@ export function ShareLink({
   const onSubmit: FormEventHandler<HTMLFormElement> = (e) => {
     e.preventDefault();
 
-    if (isPosting || isLoading) {
+    if (isPosting || isPending || isLoading) {
```

Comment:
Maybe is better to rename `isPending` and `isLoading` for give some context

Reply:
Makes sense, let me update these ones.